.gitmodules

```diff
@@ -218,7 +218,7 @@
 	url = https://github.com/ClickHouse/libpqxx
 [submodule "contrib/sqlite-amalgamation"]
 	path = contrib/sqlite-amalgamation
-	url = https://github.com/ClickHouse/sqlite-amalgamation
+	url = https://github.com/evillique/sqlite-amalgamation
```

Comment:
For testing purposes until https://github.com/ClickHouse/sqlite-amalgamation/pull/2 is merged.